.github/workflows/wheels.yml

```diff
@@ -71,6 +74,96 @@ jobs:
           name: dist
           path: ./wheelhouse/*.whl
 
+  windows:
+    name: Windows ${{ matrix.arch }}
+    runs-on: windows-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - arch: x86
+            cibw_arch: x86
+          - arch: x64
+            cibw_arch: AMD64
+          - arch: ARM64
+            cibw_arch: ARM64
+    steps:
+      - uses: actions/checkout@v4
+      - name: Checkout extra test images
+        uses: actions/checkout@v4
+        with:
+          repository: python-pillow/test-images
+          path: Tests\test-images
+      - uses: actions/setup-python@v4
+        with:
+          python-version: "3.11"
+      - name: Prepare for build
+        run: |
+          choco install nasm --no-progress
+          echo "C:\Program Files\NASM" >> $env:GITHUB_PATH
+
+          # Install extra test images
+          xcopy /S /Y Tests\test-images\* Tests\images
+
+          & python.exe -m pip install cibuildwheel==2.16.2
```

Comment:
Unfortunately not, `call winbuild\\build\\build_env.cmd` needs to be run before (but in the same step as) `%pythonLocation%\python.exe -m cibuildwheel . --output-dir wheelhouse` to set up environment variables, since `CIBW_BEFORE_ALL` does not update them.

Reply:
That's okay, we can put it in a requirements file and have Renovate update that instead: 

* https://github.com/python-pillow/Pillow/pull/7581